src/api/ADempiere/form/notice.js

```diff
@@ -48,17 +46,22 @@ export function acknowledgeNotice({
  */
 export function listUsers() {
   return request({
-    url: `${config.noticeManagement.endpoint}/users`,
+    url: `/notice-management/users`,
     method: 'get'
   })
 }
 
 /**
  * Delete Notices
  */
-export function deleteNotices() {
+export function deleteNotices(
+  id
```

Comment:
Please change `id` param name to `userId`, gives the impression that it is the id of the news and not of the user due to the name of the method, and change the name of the method to `deleteAllNotices`.